actions/next-semver/index.ts

```diff
@@ -0,0 +1,59 @@
+// noinspection DuplicatedCode
```

Comment:
The developer in me says it's probably a good idea to have tests for the functions in this file.

Reply:
I'll add some tests